<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-FUG8PLT" ignored="0" name="Test" tests="1" failures="1" timestamp="2020-09-11T14:14:07 IST" time="946.356" errors="0">
  <testcase name="feature" time="942.447" classname="cucumber.api.testng.AbstractTestNGCucumberTests">
    <failure type="cucumber.runtime.CucumberException" message="java.lang.AssertionError: expected [9.44] but found [22.857142857142858]">
      <![CDATA[cucumber.runtime.CucumberException: java.lang.AssertionError: expected [9.44] but found [22.857142857142858]
at cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:69)
at cucumber.api.testng.AbstractTestNGCucumberTests.feature(AbstractTestNGCucumberTests.java:21)
at java.util.ArrayList.forEach(Unknown Source)
Caused by: java.lang.AssertionError: expected [9.44] but found [22.857142857142858]
at com.onlineDashboardApplication.stepDefination.TestDashboard.Table_track_report_matches_couses_values_from_slp_report(TestDashboard.java:758)
at âœ½.Then Table track report matches couses values from slp report(src/test/resources/features/03_Dashboard.feature:181)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- feature -->
</testsuite> <!-- Test -->
